tests/modules/rabbitmq_example/src/Form/EsuForm.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\rabbitmq_example\Form\ContributeForm.
+ */
+
+namespace Drupal\rabbitmq_example\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Contribute form.
+ */
+class EsuForm extends FormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'rabbitmq_example_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $form['email'] = [
+      '#type' => 'email',
+      '#title' => $this->t('Your .com email address.'),
+    ];
+    $form['show'] = [
+      '#type' => 'submit',
+      '#value' => $this->t('Submit'),
+    ];
+    return $form;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function validateForm(array &$form, FormStateInterface $form_state) {
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function submitForm(array &$form, FormStateInterface $form_state) {
+    drupal_set_message($this->t('Your email address is @email', ['@email' => $form_state->getValue('email')]));
+
+    $queue_factory = \Drupal::service('queue');
```

Comment:
How are the settings managed to make sure this is loading a rabbit mq and not a database mq? If this can't part of the example module, it should at least be documented in the REAMDE of the example module